assets/src/design-system/components/menu/menu.js

```diff
@@ -77,13 +84,16 @@ const Menu = ({
     options,
     listRef,
     onDismissMenu,
+    handleReturnToParent,
   });
 
   useEffect(() => {
-    const listEl = listRef.current;
```

Comment:
Something that helped us a lot on my previous client was to add a little docstring in an effect. It helped me with:
- keeping `useEffects` simple. Long and complicated effects led to weird side effects that were difficult to debug
- making sure the code in a `useEffect` is all related. Sometimes stepping back and asking myself 'Does this belong here, or should it be split out into another effect?' helped a lot
-  adding context for people that may need to edit the effect later

Docstrings aren't a necessity, but I found some success with them 🤷